app/models/event.rb

```diff
@@ -154,6 +154,15 @@ def compute_occupied_places
     application_letters.where(status: ApplicationLetter.statuses[:accepted]).count
   end
 
+  # locks the ability to change application statuses
+  def lock_application_status
+    update(application_status_locked: true)
+  end
+  # unlocks the ability to change application statuses
+  def unlock_application_status
```

Comment:
Although, Tobi requested to be extensible in the original issue so that the locking could be removed, if you'd like to transfer sb from "Nachrücker" to "Angenommen". In this case, that method would be helpful.